server/route.go

```diff
@@ -1038,7 +1038,12 @@ func (c *client) sendRouteSubOrUnSubProtos(subs []*subscription, isSubProto, tra
 			// the lock, which could cause pingTimer to think that this
 			// connection is stale otherwise.
 			c.last = time.Now()
-			c.flushOutbound()
+			if !c.flushOutbound() {
+				// Another go routine is flushing already and does not
+				// have the lock. Give it a chance to finish...
+				c.mu.Unlock()
```

Comment:
Add a sched here? Its a no-op but just for clarity?

Reply:
Will remove that completely. This PR was based off another that was created long before I added this logic directly in `flushOutbound()`. So will clean that up.